benefits/pension_scheme.md

```diff
@@ -12,7 +12,7 @@ We provide a pension scheme inline with the UK Government's rollout of automatic
 
 As your employer we will contribute towards your pension alongside your own contribution. 
 
-From April 2019 the minimum amount that you pay into your pension is **5%** and we pay **3%** — a total minimum contribution equivalent to **8%** of your salary.
+From April 2019 the minimum amount that you pay into your pension is **5%** and we pay **3%** — a total minimum contribution equivalent to **8%** overall. All contributions are capped on qualifying earnings of £50,270 per annum. This means your overll contributions of 8% will be on full salary up to a salary of £50,270 p.a. - any salary amount above £50,270 will not increase contributions.
```

Comment:
Typo in _This means your overll_ contributions, should be _overall_?

Reply:
Isn't this on qualifying earnings i.e. earnings between £6,240 and £50,270? So your contributions would be capped at 8% of £44,030?

Basing that on https://www.gov.uk/workplace-pensions/what-you-your-employer-and-the-government-pay

Reply:
Yes, you are right @baarkerlounger. Spotted that Friday evening. I'll need to make a further update